app/app.go

```diff
@@ -203,12 +235,20 @@ func (a *App) PushHandler(push *msg.Push) {
 	}
 }
 
-// getLocalChain returns an instance of the blockchain.
-func getLocalChain() *blockchain.BlockChain {
-	// TODO: Look for local chain on disk. If doesn't exist, go rummaging
+// createBlockchain returns a new instance of a blockchain with only a genesis
+// block.
+func createBlockchain(user *User) *blockchain.BlockChain {
```

Comment:
Not invalid, just a genesis block, so it's valid. I got rid of `getLocalChain` because it was trying to do 2 things (create new blockchain or load one from disk). I'm breaking that into two functions, one is `createBlockchain` and the other will come when we support loading from disk.